src/psycop/projects/t2d/paper_outputs/model_description/performance/plotnine_confusion_matrix.py

```diff
@@ -0,0 +1,46 @@
+import pandas as pd
+import plotnine as pn
+from psycop.common.model_evaluation.confusion_matrix.confusion_matrix import (
+    ConfusionMatrix,
+)
+from psycop.common.test_utils.str_to_df import str_to_df
+from psycop.projects.t2d.paper_outputs.config import PN_THEME
+
+
+def plotnine_confusion_matrix(matrix: ConfusionMatrix, x_title: str) -> pn.ggplot:
+    df = str_to_df(
+        f"""true,pred,estimate
+Yes,+,{matrix.true_positives},
+Yes,-,{matrix.false_negatives},
+No,+,{matrix.false_positives},
+No,-,{matrix.true_negatives},
+" ",+,"PPV:\n{round(matrix.ppv, 2)}",
+" ",-,"NPV:\n{round(matrix.npv, 2)}",
+No," ","Spec:\n{round(matrix.specificity, 2)}",
+Yes," ","Sens:\n{round(matrix.sensitivity, 2)}",
+""",
+    )
+
+    """Create a confusion matrix and return a plotnine object."""
+    df["true"] = pd.Categorical(df["true"], ["Yes", "No", " "])
+    df["pred"] = pd.Categorical(df["pred"], ["+", "-", " "])
+
+    p = (
+        pn.ggplot(df, pn.aes(x="true", y="pred", fill="estimate"))
```

Comment:
```suggestion
        pn.ggplot(df, pn.aes(x="pred", y="true", fill="estimate"))
```
Predictions are usually the columns, and true values the rows (see e.g. https://scikit-learn.org/stable/modules/generated/sklearn.metrics.ConfusionMatrixDisplay.html#sklearn.metrics.ConfusionMatrixDisplay)

Reply:
Seems that's quite inconsistent in general D:

https://www.google.com/search?q=confusion+matrix&tbm=isch&ved=2ahUKEwiSk93BjPz-AhUOwioKHQo_AOUQ2-cCegQIABAA&oq=confusion+matrix&gs_lcp=CgNpbWcQAzIECCMQJzIECCMQJzIFCAAQgAQyBQgAEIAEMgUIABCABDIFCAAQgAQyBQgAEIAEMgUIABCABDIFCAAQgAQyBQgAEIAEUJEHWJ4IYMoLaABwAHgAgAFNiAHRAZIBATOYAQCgAQGqAQtnd3Mtd2l6LWltZ8ABAQ&sclient=img&ei=JaZkZNLsNI6EqwGK_oCoDg&bih=2056&biw=1920

Reply:
Let me know if you have strong opinions on this, then I'll fix it for the manuscript.